dev-utils/src/commands/genesis_command.rs

```diff
@@ -23,79 +29,240 @@ pub enum GenesisCommand {
         about = "Generate new genesis.json parameters (app_hash + app_state)"
     )]
     Generate {
+        #[structopt(
+            name = "tendermint_genesis_path",
+            short,
+            long,
+            help = "Path to the Tendermint genesis.json file (e.g. ~/.tendermint/config/genesis.json)"
+        )]
+        tendermint_genesis_path: PathBuf,
```

Comment:
Can we make `tendermint_genesis_path` optional and default to `$TMHOME/config/genesis.json` or `$HOME/config/genesis.json` if `$TMHOME` is not defined?